db/seeds/index.js

```diff
@@ -1,8 +1,20 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Movies    = require('./data/movies');
+const Locations = require('./data/locations');
 
-exports.seed = async (Knex) => {
-  await Knex('movies').truncate();
-  await Knex('movies').insert(Movies);
+exports.seed = (Knex) => {
+  return Knex.raw('TRUNCATE locations_movies CASCADE')
```

Comment:
Might be good to simplify this promise chain to `await Knex.raw('TRUNCATE TABLE movies, locations, locations_movies CASCADE')`. What do you think?

Reply:
Ah yes! I was experimenting different ways of achieving that. Will update.